src/main/java/ai/elimu/web/contributions/PublishController.java

```diff
@@ -0,0 +1,23 @@
+package ai.elimu.web.contributions;
+
+import ai.elimu.entity.enums.SupportedLanguage;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/contributions/publish")
+@Slf4j
+public class PublishController {
+
+    @GetMapping
+    public String handleRequest(Model model) {
+        log.info("handleRequest");
+
+        model.addAttribute("supportedLanguage", SupportedLanguage.values());
```

Comment:
```suggestion
        model.addAttribute("supportedLanguage", Language.values());
```

Use the existing `Language` enum instead.

Reply:
It seems that the Language enum doesn't exist within the webapp module.
Would it be okay if I rename the SupportedLanguage enum I added to Language?